src/Microsoft.DotNet.Wpf/test/XAML/DRT/DrtXaml/XamlTestClasses.FriendWithKey/XamlTestClasses.FriendWithKey.csproj

```diff
@@ -1,8 +1,11 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk" ToolsVersion="Current">
   <PropertyGroup>
     <AssemblyName>XamlTestClasses.FriendWithKey</AssemblyName>
     <RootNamespace>XamlTestClasses.FriendWithKey</RootNamespace>
     <Platforms>AnyCPU;x64</Platforms>
     <OutputType>Library</OutputType>
   </PropertyGroup>
-</Project>
+  <ItemGroup>
+    <InternalsVisibleTo Include="XamlTestClasses" />
```

Comment:
This is actually new to me (that Arcade supports this) and generally pretty cool.

Reply:
yeah it is nice!! i totally found it out on accident haha